src/hydro_system.hpp

```diff
@@ -939,4 +939,4 @@ void HydroSystem<problem_t>::ComputeFluxes(amrex::MultiFab &x1Flux_mf, amrex::Mu
 	});
 }
 
-#endif // HYDRO_SYSTEM_HPP_
+#endif // HYDRO_SYSTEM_HPP_
```

Comment:
Can you revert this change? There's no reason to modify this file.

Reply:
You can checkout the last commit that modified this file with
```
git checkout `git --no-pager log -1 --pretty=tformat:"%h" -- src/hydro_system.hpp` -- src/hydro_system.hpp
```

Reply:
Ah, oops, that won't work. This should checkout the previous version of the file:
```
git checkout `git --no-pager log -1 --skip 1 --pretty=tformat:"%h" -- src/hydro_system.hpp` -- src/hydro_system.hpp
```

Reply:
Hm, ok, the previous commit doesn't seem to be the same as that on the development branch. It might be better to just checkout the version of this file from the development branch, e.g.:
```
git checkout development -- src/hydro_system.hpp
```